NDTensors/ext/NDTensorCUDA/adapt.jl

```diff
@@ -0,0 +1,30 @@
+#
+# Used to adapt `EmptyStorage` types
+#
+
+struct NDTensorCuArrayAdaptor{B} end
```

Comment:
Could you write a comment about why this type was needed? I remember there was some behavior with handling types with some unspecified parameters that CUDA's adaptor wasn't handling properly, but it would be good to make a reminder about it.